tests/integration/connect-test.js

```diff
@@ -38,6 +38,18 @@ test('should render parent component with one state and child component with ano
   assert.equal($child.text(), 7);
 });
 
+test('should render attrs', function(assert) {
+  assert.expect(2);
+  this.set('myName', 'Dustin');
+  this.render(hbs`{{count-list name=myName}}`);
+
+  assert.equal(this.$('.greeting').text(), 'Welcome back, Dustin!', 'should render attrs provided to component');
+
+  this.set('myName', 'Toran');
+
+  assert.equal(this.$('.greeting').text(), 'Welcome back, Toran!', 'should rerender component if attrs change');
```

Comment:
love the simplicity of this test case - great work on this pull request Dustin!!!! Truly grateful to have someone like you contributing to this project!